src/oas2/transformers/servers.ts

```diff
@@ -20,8 +20,8 @@ export function translateToServers(spec: Partial<Spec>, operation: Partial<Opera
       .scheme(scheme)
       .host(host);
 
-    if (basePath) {
-      uri = uri.path(basePath);
+    if (typeof basePath === 'string' && basePath.length > 0) {
+      uri = uri.path(String(basePath));
```

Comment:
If `basePath` is already a string, why the cast here?

Reply:
```suggestion
      uri = uri.path(basePath);
```
Yeah, it's a leftover, as I hadn't had the check before I added it.